src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -1019,7 +1019,8 @@ public RichCustomEmojiImpl createEmoji(GuildImpl guildObj, DataObject json)
     public ApplicationEmojiImpl createApplicationEmoji(JDAImpl api, DataObject json)
     {
         final long emojiId = json.getLong("id");
-        final User user = createUser(json.getObject("user"));
+        // Nullable when creating emojis, at least
+        final User user = json.optObject("user").map(this::createUser).orElse(null);
```

Comment:
When exactly is this nullable?
It shouldn't be nullable given it afaik always has the user.

Reply:
The user object is absent from the data returned when you create an application emoji, it is nullable in the API docs too